nexus-webapi/tests/user/bootstrap.rs

```diff
@@ -18,6 +18,8 @@ async fn test_bootstrap_user() -> Result<()> {
     assert_eq!(user_bootstrap_respose.list.stream.len(), 20);
     assert_eq!(user_bootstrap_respose.list.influencers.len(), 3);
     assert_eq!(user_bootstrap_respose.list.recommended.len(), 5);
+    println!("hot_tags: {:?}", user_bootstrap_respose.list.hot_tags);
```

Comment:
nip: Is this println necessary? Sorry to ask, If you guys are using console.logs in test, okay!

```suggestion
```